### DISCUSSION TITLE
Feedback for dependabot's `recreate` command

### DISCUSSION THREAD

**Discussion by spenserblack** [community/community#27734] [Code Security]
It seems like, for a new PR with no additional edits, `recreate` and `rebase` are the same. One thing that surprised me is that `recreate` does *not* fetch a dependency's release notes again. This was surprising behavior for me, because there was an instance where a dependency's release notes were updated, but I didn't see the updates added to the PR after commenting `@dependabot recreate`. My interpretation of "recreate" is that *everything* is recreated, and I'd expect the process for creating a PR and *re*creating a PR (such as fetching release notes) to be almost identical. But, if I'm not mistaken, `recreate` really *only* recreates the initial commit for the PR, erasing additional commits.

To summarize, `recreate` seems to be only concerned with the PR's commit history, but I expected `recreate` to also be effective for the initial PR comment, too.
